docs/source/includes/install_from_pypi.rst

```diff
@@ -1,11 +1,10 @@
-Please run the following command to install the required dependencies
+Please run the `install script <https://github.com/luxonis/depthai-python/blob/main/examples/install_requirements.py>`__
```

Comment:
Need to mention that this script must be run from git context, after depthai-python is cloned, it's not enough to download this script only.